scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -25,7 +25,7 @@ workflow GvsExtractCallset {
 
     File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
     Boolean use_interval_weights = true
-    File interval_weights_bed = "gs://broad-public-datasets/gvs/weights/gvs_vet_weights_1kb.bed"
+    File interval_weights_bed = "gs://gvs_quickstart_storage/weights/gvs_full_vet_weights_1kb_padded.bed"
```

Comment:
interval_weights_bed is also passed as an argument from GvsJointVariantCalling.wdl - now that we have one standard bed file, you probably can remove the optional File variable there?

Reply:
I missed that completely, thanks!  I've updated the path in GVSJointVariantCalling, and I'll take your advice about removing the argument there.  We MIGHT, if we expend to handle non-human genomes or different references, want the ability to add in a new file in the future.  But I'd much rather we just add it back in at that time if we decide we need it rather than leave an optional argument around just in case